src/main/java/org/linecode/server/business/UnitStatus.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * PORTACS
+ * piattaforma di controllo mobilità autonoma
+ *
+ * Copyright lineCode group <linecode.swe@gmail.com> 2020 - 2021
+ * Distributed under open-source licence (see accompanying file LICENCE).
+ */
+package org.linecode.server.business;
+
+public enum UnitStatus {
+    GOINGTO(0),STOP(1),BASE(2),ERROR(3),DISCONNECTED(4);
+
+    private Integer code;
+    UnitStatus(final Integer code) {
+        this.code=code;
+    }
+    public Integer getCode(){
+        return code;
+    }
```

Comment:
In questo modo viene "aggirato" l'utilizzo dell'enum salvando un intero che poi viene richiamato tale e quale. L'enum lega naturalmente dei numeri naturali ad ogni sua costante ed è possibile lavorare tramite essi. Fonte breve ma esaustiva: https://www.javatpoint.com/enum-in-java. Da correggere anche sul diagramma.